flow/project.py

```diff
@@ -3769,10 +4082,25 @@ def _main_submit(self, args):
         self.submit(jobs=jobs, names=names, **kwargs)
 
     def _main_exec(self, args):
-        if len(args.jobid):
-            jobs = [self.open_job(id=jid) for jid in args.jobid]
+        if args.jobid:
+            jobs = set()
+            for _id in args.jobid:
+                try:
+                    if self.open_job(id=_id):
+                        jobs.add(_id)
+                except KeyError as error:
+                    # TODO: The below commented code is to be included when we
+                    # introduce actual aggregation.
+                    # if "agg-" == id[0:4]:
+                    #     jobs.add(id)
+                    raise LookupError("Did not find job with id {}.".format(error))
+
+            self._verify_aggregate_project(deepcopy(jobs))
```

Comment:
This was because we were checking whether the aggregates are valid or not.
We were deleting items from the list too.
I believe we don't need this anymore